content/applications/inventory_and_mrp/maintenance/maintenance_requests.rst

```diff
@@ -0,0 +1,98 @@
+:guilabel:`Equipment` or :guilabel:`Work Center`. Using the drop-down menu for either field, select
+a piece of equipment or a work center.
+
+The next field is titled :guilabel:`Request Date`, and is set by default to the date on which the
+maintenance request is created. This date cannot be changed by the user.
+
+In the :guilabel:`Maintenance` field, select the :guilabel:`Corrective` option if the request is
+intended to fix an existing issue, or the :guilabel:`Preventive` option if the request is intended
+to prevent issues from occurring in the future.
+
+If the request is being created to address an issue that arose during a specific manufacturing order
+(MO), select it in the :guilabel:`Manufacturing Order` field.
+
+If an |MO| was selected in the :guilabel:`Manufacturing Order` field, a :guilabel:`Work Order` field
+appears below it. If the issue arose during a specific work order, specify it in this field.
+
+In the :guilabel:`Team` field, select the maintenance team that is responsible for managing the
+request. If a specific team member is responsible, select them in the :guilabel:`Responsible` field.
+
+The :guilabel:`Scheduled Date` field is used to specify the date on which maintenance should take
+place, and the time it should begin. Choose a date by clicking on the field to open a calendar in a
+pop-up window, and then select a day on the calendar. Enter an hour and minute in the two fields
+below the calendar, and click :guilabel:`Apply` to save the date and time.
+
+The :guilabel:`Duration` field is used to specify the time it takes to complete the maintenance
+request. Use the text-entry field to enter the time in a `00:00` format.
+
+If :guilabel:`Work Center` was selected in the :guilabel:`For` field, a :guilabel:`Block Workcenter`
+checkbox appears below the :guilabel:`Duration` field. Enable the checkbox to prevent work orders or
+other maintenance from being scheduled at the specified work center while the maintenance request is
+being processed.
+
+The :guilabel:`Priority` field is used to communicate the importance or urgency of the maintenance
+request. Assign the request a priority between zero and three stars, by clicking on the desired star
+number. Requests assigned a higher priority appear above those with a lower priority, on the Kanban
+board used to track the progression of maintenance requests.
+
+In the :guilabel:`Notes` tab at the bottom of the form, enter any relevant details about the
+maintenance request (why the maintenance issue arose, when it occurred, etc.).
+
+The :guilabel:`Instructions` tab is used to include instructions for how maintenance should be
+performed. Select one of the three options, and then include the instructions as detailed below:
+
+- :guilabel:`PDF`: click the :guilabel:`Upload your file` button to open the device's file manager,
+  and then select a file to upload.
+- :guilabel:`Google Slide`: enter a Google Slide link in the text-entry field that appears after the
```

Comment:
```suggestion
- :guilabel:`Google Slide`: enter a :guilabel:`Google Slide link` in the text-entry field that appears after the
```
using a GUI here because that's how it actually appears on the UI